src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -97,6 +97,95 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: GET
 
+
+  #
+  # Temporary Certificate Explainer Document
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/temporary-certificate/{gssCode}/explainer-document':
```

Comment:
Possibly, can move the `gssCode` to  a request body - included in the URL as initially thought this would be a `GET`